src/sonic-yang-models/yang-models/sonic-vlan.yang

```diff
@@ -102,6 +102,16 @@ module sonic-vlan {
 					}
 				}
 
+				leaf ipv6_use_link_local_only {
+					description "Enable/Disable IPv6 link local address on vlan interface";
+					type enumeration {
```

Comment:
How about this type?
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L178

Reply:
Done.